parama/index.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: VilniusRB 2% Parama
+---
+
+<div class="row">
+  <div class="large-12 columns">
+    <p>
+      The following information is for Lithuanians who get income and
+      can dedicate 2% from it to any selected non profit organization.
+    </p>
+
+    <h4>Prašymas pervesti iki 2 procentų pajamų mokesčio "VilniusRB" organizacijai.</h4>
+    <p>
+      Jei turėjote apmokestinamų pajamų, tuomet kiekvienais metais iki gegužės 1 d.
```

Comment:
As galvoju, ar reikia tokio pilno isaiskinimo? 

Ar ko nors paprastesnio, tarkim kazko tokio: 

"Kviečiame skirti iki 2% sumokėto gyventojų pajamų mokesčio VilniusRB susitikimų vystymui

Jeigu jums patinka ką mes darome, kviečiame paremti VšĮ "Rubistai" skiriant dalį sumokėto pajamų mokesčio.
*Visos* gautos lėšos bus panaudotos kalbėtojams atsivežti!"